internal/usecase/interactor/auth.go

```diff
@@ -1,4 +1,4 @@
-package appauth
```

Comment:
`interactor.Storage` is too abstract name and I don't know what the storage is in the interactor. `interactor.AuthStorage` is ok.